services/minespace-web/src/components/Forms/projects/informationRequirementsTable/ReviewSubmitInformationRequirementsTable.js

```diff
@@ -10,11 +10,15 @@ const propTypes = {
   sub_requirement: customPropTypes.subRequirements.isRequired,
 };
 
-const renderSubrequirement = (item) => item.map((it) => <Subrequirement sub_requirement={it} />);
+const renderSubrequirement = (item) =>
+  item.map((it) => {
+    //eslint-disable-next-line @typescript-eslint/no-use-before-define
```

Comment:
all I wanted to do in this file was add a bordered=false param below, but eslint was not happy about that rule, and I found that the logic was kind of circular. I _did_ give it its missing key though.